### PR TITLE
Update testnet tutorial

### PR DESCRIPTION
**Pull request by toschdev**
Update testnet tutorial with latest changes to the `cosmoshub-testnet`

### CODE REVIEW

**Review by helder-moreira — APPROVED** [review]
LGTM Thanks!

File: connecting-to-testnet/index.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ description: Start a node, use the Cosmos SDK CLI, and interact with the Cosmos
...
 
 ## What is a testnet?
 
 A testnet is a network launched for developers and validators to test the functionality of the blockchain before the main network is launched. The state of a testnet can be reset at any time.
 
-## Tutorial Testnet 
+## Tutorial Testnet
 
-The source code repo `b-harvest/gravity-dex` offers more functionality because it adds the `tendermint/liquidity` module that is the source code of Gravity DEX to the `cosmos/gaia` source code behind Cosmos Hub. 
+The source code repo `b-harvest/gravity-dex` offers more functionality because it adds the `tendermint/liquidity` module that is the source code of Gravity DEX to the `cosmos/gaia` source code behind Cosmos Hub.
```
Comment (barriebyron) [inline connecting-to-testnet/index.md:20]: @toschdev I see how you updated the content to point to gaia... does line 20 still make sense? (how would I know? teach me!)

**Review by barriebyron — COMMENTED** [review]
@toschdev could you walk me through how to verify these technical details? I am teachable!

File: connecting-to-testnet/index.md (hunk 1 above)
Comment (toschdev) [reply connecting-to-testnet/index.md]: Thanks, overlooked this occurence.

**Comment by toschdev** [comment]
In need of one more approval

**Review by barriebyron — APPROVED** [review]
thanks @toschdev !

**Pull request merged**